submissions/nhunzaker/test/server.test.js

```diff
@@ -0,0 +1,42 @@
+var Server = require('../lib/server')
+var assert = require('assert')
+
+describe('Server', function() {
+
+  context('When Server.init is called at a given port', function() {
+    var PORT = 9000
+
+    beforeEach(function (next) {
+      this.server = Server.init(PORT, next)
```

Comment:
Any reason to prefer attaching shared things to `this` versus just binding to a variable higher up in scope?